apps/gallery/js/thumbnails.js

```diff
@@ -18,13 +18,7 @@
   // contained in the thumbnailContainer.
   thumbnails.list = new ThumbnailList(ThumbnailDateGroup, thumbnails.container);
 
-  // Localize dates and item desc in thumbnails list when the locale changes
-  // Bug 1135256: Having a listener in ThumbnailList prevents them from
-  // being garbage collected since ready() holds a strong reference, so
-  // we have a singleton listener here.
-  navigator.mozL10n.ready(function localize() {
-    thumbnails.list.localize();
-  });
+  navigator.mozL10n.ready(thumbnails.list.localize.bind(thumbnails.list));
```

Comment:
nit: I understand even with l10n API, Bug 1135256 is still there and we should keep comment from Line 21-24 explaining why we are not moving this listener to ThumbnailList